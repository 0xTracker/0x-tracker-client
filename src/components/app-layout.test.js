import React from 'react';

import { renderWithAppContext } from '../test-util/react';
import AppLayout from './app-layout';

describe('app layout component', () => {
  it('should render with children', () => {
    const { container } = renderWithAppContext(
      <AppLayout>Hello World</AppLayout>,
    );

    expect(container.firstChild).toMatchSnapshot();
  });
});
