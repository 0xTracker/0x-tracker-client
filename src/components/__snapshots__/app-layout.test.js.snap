// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app layout component should render with children 1`] = `
<div
  class="app-layout__AppContainer-nr5sez-0 iAFouo"
>
  <div
    class="sidebar___StyledDiv-sc-1ginpey-3 iLqFtO"
  >
    <a
      class="link__StyledLink-sc-1m1n17q-0 diqghp sidebar___StyledLink-sc-1ginpey-4 glyHCt"
      href="/"
    >
      <img
        alt="0x Tracker"
        class="sidebar__LogoImage-sc-1ginpey-0 kYAoTy"
        src="test-file-stub"
      />
    </a>
    <nav
      aria-label="Primary"
      class="navigation__StyledNavigation-sc-18grpkd-0 jAleuT"
    >
      <a
        aria-current="false"
        class="link__StyledLink-sc-1m1n17q-0 diqghp navigation-link__StyledNavigationLink-do5zl-0 dPHgzV"
        href="/apps"
      >
        <svg
          aria-hidden="true"
          class="icon-base__IconBase-ngmsqy-0 hvjdWy navigation__NavigationIcon-sc-18grpkd-1 hWVwlC"
          fill="currentColor"
          focusable="false"
          height="24"
          viewBox="0 0 512 512"
          width="24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M104 160a56 56 0 1156-56 56.06 56.06 0 01-56 56zM256 160a56 56 0 1156-56 56.06 56.06 0 01-56 56zM408 160a56 56 0 1156-56 56.06 56.06 0 01-56 56zM104 312a56 56 0 1156-56 56.06 56.06 0 01-56 56zM256 312a56 56 0 1156-56 56.06 56.06 0 01-56 56zM408 312a56 56 0 1156-56 56.06 56.06 0 01-56 56zM104 464a56 56 0 1156-56 56.06 56.06 0 01-56 56zM256 464a56 56 0 1156-56 56.06 56.06 0 01-56 56zM408 464a56 56 0 1156-56 56.06 56.06 0 01-56 56z"
          />
        </svg>
         Apps
      </a>
      <a
        aria-current="false"
        class="link__StyledLink-sc-1m1n17q-0 diqghp navigation-link__StyledNavigationLink-do5zl-0 dPHgzV"
        href="/fills"
      >
        <svg
          aria-hidden="true"
          class="icon-base__IconBase-ngmsqy-0 hvjdWy navigation__NavigationIcon-sc-18grpkd-1 hWVwlC"
          fill="currentColor"
          focusable="false"
          height="24"
          viewBox="0 0 24 24"
          width="24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M0 0h24v24H0z"
            fill="none"
          />
          <path
            d="M3 3h18a1 1 0 011 1v16a1 1 0 01-1 1H3a1 1 0 01-1-1V4a1 1 0 011-1zm12 4v2h-4v2h4v2l3.5-3L15 7zM9 17v-2h4v-2H9v-2l-3.5 3L9 17z"
          />
        </svg>
         Browse Trades
      </a>
      <a
        aria-current="false"
        class="link__StyledLink-sc-1m1n17q-0 diqghp navigation-link__StyledNavigationLink-do5zl-0 dPHgzV"
        href="/network-insights"
      >
        <svg
          aria-hidden="true"
          class="icon-base__IconBase-ngmsqy-0 hvjdWy navigation__NavigationIcon-sc-18grpkd-1 hWVwlC"
          fill="currentColor"
          focusable="false"
          height="24"
          viewBox="0 0 512 512"
          width="24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M332.8 320h38.4c6.4 0 12.8-6.4 12.8-12.8V172.8c0-6.4-6.4-12.8-12.8-12.8h-38.4c-6.4 0-12.8 6.4-12.8 12.8v134.4c0 6.4 6.4 12.8 12.8 12.8zm96 0h38.4c6.4 0 12.8-6.4 12.8-12.8V76.8c0-6.4-6.4-12.8-12.8-12.8h-38.4c-6.4 0-12.8 6.4-12.8 12.8v230.4c0 6.4 6.4 12.8 12.8 12.8zm-288 0h38.4c6.4 0 12.8-6.4 12.8-12.8v-70.4c0-6.4-6.4-12.8-12.8-12.8h-38.4c-6.4 0-12.8 6.4-12.8 12.8v70.4c0 6.4 6.4 12.8 12.8 12.8zm96 0h38.4c6.4 0 12.8-6.4 12.8-12.8V108.8c0-6.4-6.4-12.8-12.8-12.8h-38.4c-6.4 0-12.8 6.4-12.8 12.8v198.4c0 6.4 6.4 12.8 12.8 12.8zM496 384H64V80c0-8.84-7.16-16-16-16H16C7.16 64 0 71.16 0 80v336c0 17.67 14.33 32 32 32h464c8.84 0 16-7.16 16-16v-32c0-8.84-7.16-16-16-16z"
            fill="currentColor"
          />
        </svg>
         Network Insights
      </a>
      <a
        aria-current="false"
        class="link__StyledLink-sc-1m1n17q-0 diqghp navigation-link__StyledNavigationLink-do5zl-0 dPHgzV"
        href="/news-and-updates"
        title="News & Updates"
      >
        <svg
          aria-hidden="true"
          class="icon-base__IconBase-ngmsqy-0 hvjdWy navigation__NavigationIcon-sc-18grpkd-1 hWVwlC"
          fill="currentColor"
          focusable="false"
          height="22"
          viewBox="0 0 576 512"
          width="22"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M552 64H88c-13.255 0-24 10.745-24 24v8H24c-13.255 0-24 10.745-24 24v272c0 30.928 25.072 56 56 56h472c26.51 0 48-21.49 48-48V88c0-13.255-10.745-24-24-24zM56 400a8 8 0 01-8-8V144h16v248a8 8 0 01-8 8zm236-16H140c-6.627 0-12-5.373-12-12v-8c0-6.627 5.373-12 12-12h152c6.627 0 12 5.373 12 12v8c0 6.627-5.373 12-12 12zm208 0H348c-6.627 0-12-5.373-12-12v-8c0-6.627 5.373-12 12-12h152c6.627 0 12 5.373 12 12v8c0 6.627-5.373 12-12 12zm-208-96H140c-6.627 0-12-5.373-12-12v-8c0-6.627 5.373-12 12-12h152c6.627 0 12 5.373 12 12v8c0 6.627-5.373 12-12 12zm208 0H348c-6.627 0-12-5.373-12-12v-8c0-6.627 5.373-12 12-12h152c6.627 0 12 5.373 12 12v8c0 6.627-5.373 12-12 12zm0-96H140c-6.627 0-12-5.373-12-12v-40c0-6.627 5.373-12 12-12h360c6.627 0 12 5.373 12 12v40c0 6.627-5.373 12-12 12z"
            fill="currentColor"
          />
        </svg>
         News & Updates
      </a>
      <a
        aria-current="false"
        class="link__StyledLink-sc-1m1n17q-0 diqghp navigation-link__StyledNavigationLink-do5zl-0 dPHgzV"
        href="/tokens"
      >
        <svg
          aria-hidden="true"
          class="icon-base__IconBase-ngmsqy-0 hvjdWy navigation__NavigationIcon-sc-18grpkd-1 hWVwlC"
          fill="currentColor"
          focusable="false"
          height="26"
          viewBox="0 0 24 24"
          width="26"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M0 0h24v24H0z"
            fill="none"
          />
          <path
            d="M14 2a8 8 0 013.292 15.293A8 8 0 116.706 6.707 8.003 8.003 0 0114 2zm-3 7H9v1a2.5 2.5 0 00-.164 4.995L9 15h2l.09.008a.5.5 0 010 .984L11 16H7v2h2v1h2v-1a2.5 2.5 0 00.164-4.995L11 13H9l-.09-.008a.5.5 0 010-.984L9 12h4v-2h-2V9zm3-5a5.985 5.985 0 00-4.484 2.013 8 8 0 018.47 8.471A6 6 0 0014 4z"
          />
        </svg>
         Tokens
      </a>
      <a
        aria-current="false"
        class="link__StyledLink-sc-1m1n17q-0 diqghp navigation-link__StyledNavigationLink-do5zl-0 dPHgzV"
        href="/traders"
      >
        <svg
          aria-hidden="true"
          class="icon-base__IconBase-ngmsqy-0 hvjdWy navigation__NavigationIcon-sc-18grpkd-1 hWVwlC"
          fill="currentColor"
          focusable="false"
          height="26"
          viewBox="0 0 24 24"
          width="26"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M0 0h24v24H0V0z"
            fill="none"
          />
          <path
            d="M16 11c1.66 0 2.99-1.34 2.99-3S17.66 5 16 5s-3 1.34-3 3 1.34 3 3 3zm-8 0c1.66 0 2.99-1.34 2.99-3S9.66 5 8 5 5 6.34 5 8s1.34 3 3 3zm0 2c-2.33 0-7 1.17-7 3.5V18c0 .55.45 1 1 1h12c.55 0 1-.45 1-1v-1.5c0-2.33-4.67-3.5-7-3.5zm8 0c-.29 0-.62.02-.97.05.02.01.03.03.04.04 1.14.83 1.93 1.94 1.93 3.41V18c0 .35-.07.69-.18 1H22c.55 0 1-.45 1-1v-1.5c0-2.33-4.67-3.5-7-3.5z"
          />
        </svg>
         Traders
      </a>
    </nav>
    <div
      class="sidebar__Footer-sc-1ginpey-2 gKnwRH"
    >
      <button
        class="sidebar__ContractButton-sc-1ginpey-1 hMciQI"
        type="button"
      >
        <svg
          aria-hidden="true"
          class="icon-base__IconBase-ngmsqy-0 hvjdWy"
          fill="currentColor"
          focusable="false"
          height="20"
          viewBox="0 0 24 24"
          width="20"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M0 0h24v24H0z"
            fill="none"
          />
          <path
            d="M21 18v2H3v-2h18zM6.596 3.904L8.01 5.318 4.828 8.5l3.182 3.182-1.414 1.414L2 8.5l4.596-4.596zM21 11v2h-9v-2h9zm0-7v2h-9V4h9z"
          />
        </svg>
      </button>
    </div>
  </div>
  <div
    class="app-layout__Main-nr5sez-2 cBExpF"
  >
    <div
      class="app-layout__Scrollport-nr5sez-3 bxkNbZ"
    >
      <header
        class="header__StyledHeader-gflnft-1 DipzI"
      >
        <div
          class="header___StyledDiv-gflnft-3 koKNgv"
        >
          <div
            class="header___StyledDiv2-gflnft-4 jFJKTJ"
          >
            <div
              class="search-box__SearchOverlay-sc-14vl3sw-0 cHVapg"
            />
            <div
              class="card__Wrapper-sc-1u97707-0 fHGyNm search-box___StyledCard-sc-14vl3sw-1 gZIiCY"
            >
              <div
                class="card-body__CardBody-sc-1mkhz93-0 search-box___StyledCardBody-sc-14vl3sw-2 bhXCds"
              >
                <form
                  action="/search"
                  class="search-box___StyledForm-sc-14vl3sw-3 jFpRsd"
                  method="get"
                >
                  <div
                    class="search-input__InputWrapper-m0ltwv-0 EQckh"
                  >
                    <div
                      aria-expanded="false"
                      aria-haspopup="listbox"
                      aria-owns="react-autowhatever-1"
                      class="react-autosuggest__container"
                      role="combobox"
                    >
                      <input
                        aria-autocomplete="list"
                        aria-controls="react-autowhatever-1"
                        autocomplete="off"
                        class="react-autosuggest__input"
                        name="q"
                        placeholder="Search for tokens, apps, traders or trades"
                        type="text"
                        value=""
                      />
                      <div
                        class="react-autosuggest__suggestions-container"
                        id="react-autowhatever-1"
                        role="listbox"
                      />
                    </div>
                  </div>
                  <div
                    class="search-box___StyledDiv-sc-14vl3sw-4 egXxeQ"
                  >
                    <svg
                      class="search-box___StyledSvg-sc-14vl3sw-5 gZBlRd"
                      fill="currentColor"
                      focusable="false"
                      height="22"
                      role="img"
                      viewBox="0 0 576 512"
                      width="22"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M528 64H48C21.49 64 0 85.49 0 112v288c0 26.51 21.49 48 48 48h480c26.51 0 48-21.49 48-48V112c0-26.51-21.49-48-48-48zm8 336c0 4.411-3.589 8-8 8H48c-4.411 0-8-3.589-8-8V112c0-4.411 3.589-8 8-8h480c4.411 0 8 3.589 8 8v288zM170 270v-28c0-6.627-5.373-12-12-12h-28c-6.627 0-12 5.373-12 12v28c0 6.627 5.373 12 12 12h28c6.627 0 12-5.373 12-12zm96 0v-28c0-6.627-5.373-12-12-12h-28c-6.627 0-12 5.373-12 12v28c0 6.627 5.373 12 12 12h28c6.627 0 12-5.373 12-12zm96 0v-28c0-6.627-5.373-12-12-12h-28c-6.627 0-12 5.373-12 12v28c0 6.627 5.373 12 12 12h28c6.627 0 12-5.373 12-12zm96 0v-28c0-6.627-5.373-12-12-12h-28c-6.627 0-12 5.373-12 12v28c0 6.627 5.373 12 12 12h28c6.627 0 12-5.373 12-12zm-336 82v-28c0-6.627-5.373-12-12-12H82c-6.627 0-12 5.373-12 12v28c0 6.627 5.373 12 12 12h28c6.627 0 12-5.373 12-12zm384 0v-28c0-6.627-5.373-12-12-12h-28c-6.627 0-12 5.373-12 12v28c0 6.627 5.373 12 12 12h28c6.627 0 12-5.373 12-12zM122 188v-28c0-6.627-5.373-12-12-12H82c-6.627 0-12 5.373-12 12v28c0 6.627 5.373 12 12 12h28c6.627 0 12-5.373 12-12zm96 0v-28c0-6.627-5.373-12-12-12h-28c-6.627 0-12 5.373-12 12v28c0 6.627 5.373 12 12 12h28c6.627 0 12-5.373 12-12zm96 0v-28c0-6.627-5.373-12-12-12h-28c-6.627 0-12 5.373-12 12v28c0 6.627 5.373 12 12 12h28c6.627 0 12-5.373 12-12zm96 0v-28c0-6.627-5.373-12-12-12h-28c-6.627 0-12 5.373-12 12v28c0 6.627 5.373 12 12 12h28c6.627 0 12-5.373 12-12zm96 0v-28c0-6.627-5.373-12-12-12h-28c-6.627 0-12 5.373-12 12v28c0 6.627 5.373 12 12 12h28c6.627 0 12-5.373 12-12zm-98 158v-16c0-6.627-5.373-12-12-12H180c-6.627 0-12 5.373-12 12v16c0 6.627 5.373 12 12 12h216c6.627 0 12-5.373 12-12z"
                        fill="currentColor"
                      />
                    </svg>
                    /
                  </div>
                </form>
              </div>
            </div>
          </div>
          <div
            class="header-actions___StyledDiv-sc-10491ou-1 bKdmbO"
          >
            <a
              class="link__StyledLink-sc-1m1n17q-0 link___StyledStyledLink-sc-1m1n17q-1 jcGDtB header-actions__ActionButton-sc-10491ou-0 hBMyMT"
              href="https://github.com/0xTracker"
              rel="noreferrer noopener"
              target="_blank"
              title="0x Tracker on GitHub"
            >
              <svg
                aria-hidden="true"
                class="icon-base__IconBase-ngmsqy-0 hvjdWy"
                color="currentColor"
                fill="currentColor"
                focusable="false"
                height="24"
                viewBox="0 0 496 512"
                width="24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"
                  fill="currentColor"
                />
              </svg>
            </a>
            <button
              class="header-actions__ActionButton-sc-10491ou-0 hBMyMT"
              title="Settings"
            >
              <svg
                aria-hidden="true"
                class="icon-base__IconBase-ngmsqy-0 hvjdWy"
                color="currentColor"
                fill="currentColor"
                focusable="false"
                height="24"
                viewBox="0 0 512 512"
                width="24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M487.4 315.7l-42.6-24.6c4.3-23.2 4.3-47 0-70.2l42.6-24.6c4.9-2.8 7.1-8.6 5.5-14-11.1-35.6-30-67.8-54.7-94.6-3.8-4.1-10-5.1-14.8-2.3L380.8 110c-17.9-15.4-38.5-27.3-60.8-35.1V25.8c0-5.6-3.9-10.5-9.4-11.7-36.7-8.2-74.3-7.8-109.2 0-5.5 1.2-9.4 6.1-9.4 11.7V75c-22.2 7.9-42.8 19.8-60.8 35.1L88.7 85.5c-4.9-2.8-11-1.9-14.8 2.3-24.7 26.7-43.6 58.9-54.7 94.6-1.7 5.4.6 11.2 5.5 14L67.3 221c-4.3 23.2-4.3 47 0 70.2l-42.6 24.6c-4.9 2.8-7.1 8.6-5.5 14 11.1 35.6 30 67.8 54.7 94.6 3.8 4.1 10 5.1 14.8 2.3l42.6-24.6c17.9 15.4 38.5 27.3 60.8 35.1v49.2c0 5.6 3.9 10.5 9.4 11.7 36.7 8.2 74.3 7.8 109.2 0 5.5-1.2 9.4-6.1 9.4-11.7v-49.2c22.2-7.9 42.8-19.8 60.8-35.1l42.6 24.6c4.9 2.8 11 1.9 14.8-2.3 24.7-26.7 43.6-58.9 54.7-94.6 1.5-5.5-.7-11.3-5.6-14.1zM256 336c-44.1 0-80-35.9-80-80s35.9-80 80-80 80 35.9 80 80-35.9 80-80 80z"
                  fill="currentColor"
                />
              </svg>
            </button>
          </div>
        </div>
      </header>
      <div
        class="app-layout__AppBody-nr5sez-1 htrrnH"
      >
        Hello World
      </div>
      <div
        class="footer__Wrapper-htpw4v-10 ga-dXKY"
      >
        <div
          class="row"
        >
          <div
            class="footer__Info-htpw4v-9 cFMPKE col-12 col-md-5 col-lg-4"
          >
            <h2
              class="footer__InfoHeading-htpw4v-0 cqvhbg"
            >
              0x Tracker
            </h2>
            <p
              class="footer__Description-htpw4v-2 fPIQvA"
            >
              The leading provider of 0x protocol market data, transparent Ethereum token price index and 0x protocol news aggregator.
            </p>
            <nav
              class="footer___StyledNav-htpw4v-11 hhjFbS"
            >
              <a
                class="link__StyledLink-sc-1m1n17q-0 link___StyledStyledLink-sc-1m1n17q-1 jcGDtB footer__SocialLink-htpw4v-5 fyprxL"
                href="https://twitter.com/0xTracker"
                rel="noreferrer noopener"
                target="_blank"
              >
                <svg
                  aria-hidden="true"
                  class="icon-base__IconBase-ngmsqy-0 hvjdWy"
                  fill="currentColor"
                  focusable="false"
                  height="18"
                  viewBox="0 0 512 512"
                  width="18"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M459.37 151.716c.325 4.548.325 9.097.325 13.645 0 138.72-105.583 298.558-298.558 298.558-59.452 0-114.68-17.219-161.137-47.106 8.447.974 16.568 1.299 25.34 1.299 49.055 0 94.213-16.568 130.274-44.832-46.132-.975-84.792-31.188-98.112-72.772 6.498.974 12.995 1.624 19.818 1.624 9.421 0 18.843-1.3 27.614-3.573-48.081-9.747-84.143-51.98-84.143-102.985v-1.299c13.969 7.797 30.214 12.67 47.431 13.319-28.264-18.843-46.781-51.005-46.781-87.391 0-19.492 5.197-37.36 14.294-52.954 51.655 63.675 129.3 105.258 216.365 109.807-1.624-7.797-2.599-15.918-2.599-24.04 0-57.828 46.782-104.934 104.934-104.934 30.213 0 57.502 12.67 76.67 33.137 23.715-4.548 46.456-13.32 66.599-25.34-7.798 24.366-24.366 44.833-46.132 57.827 21.117-2.273 41.584-8.122 60.426-16.243-14.292 20.791-32.161 39.308-52.628 54.253z"
                    fill="currentColor"
                  />
                </svg>
              </a>
              <a
                class="link__StyledLink-sc-1m1n17q-0 link___StyledStyledLink-sc-1m1n17q-1 jcGDtB footer__SocialLink-htpw4v-5 fyprxL"
                href="https://discord.gg/tnV8hud"
                rel="noreferrer noopener"
                target="_blank"
              >
                <svg
                  aria-hidden="true"
                  class="icon-base__IconBase-ngmsqy-0 hvjdWy"
                  fill="currentColor"
                  focusable="false"
                  height="18"
                  viewBox="0 0 448 512"
                  width="18"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M297.216 243.2c0 15.616-11.52 28.416-26.112 28.416-14.336 0-26.112-12.8-26.112-28.416s11.52-28.416 26.112-28.416c14.592 0 26.112 12.8 26.112 28.416zm-119.552-28.416c-14.592 0-26.112 12.8-26.112 28.416s11.776 28.416 26.112 28.416c14.592 0 26.112-12.8 26.112-28.416.256-15.616-11.52-28.416-26.112-28.416zM448 52.736V512c-64.494-56.994-43.868-38.128-118.784-107.776l13.568 47.36H52.48C23.552 451.584 0 428.032 0 398.848V52.736C0 23.552 23.552 0 52.48 0h343.04C424.448 0 448 23.552 448 52.736zm-72.96 242.688c0-82.432-36.864-149.248-36.864-149.248-36.864-27.648-71.936-26.88-71.936-26.88l-3.584 4.096c43.52 13.312 63.744 32.512 63.744 32.512-60.811-33.329-132.244-33.335-191.232-7.424-9.472 4.352-15.104 7.424-15.104 7.424s21.248-20.224 67.328-33.536l-2.56-3.072s-35.072-.768-71.936 26.88c0 0-36.864 66.816-36.864 149.248 0 0 21.504 37.12 78.08 38.912 0 0 9.472-11.52 17.152-21.248-32.512-9.728-44.8-30.208-44.8-30.208 3.766 2.636 9.976 6.053 10.496 6.4 43.21 24.198 104.588 32.126 159.744 8.96 8.96-3.328 18.944-8.192 29.44-15.104 0 0-12.8 20.992-46.336 30.464 7.68 9.728 16.896 20.736 16.896 20.736 56.576-1.792 78.336-38.912 78.336-38.912z"
                    fill="currentColor"
                  />
                </svg>
              </a>
              <a
                class="link__StyledLink-sc-1m1n17q-0 diqghp footer__SocialLink-htpw4v-5 fyprxL"
                href="mailto:hello@0xtracker.com"
              >
                <svg
                  aria-hidden="true"
                  class="icon-base__IconBase-ngmsqy-0 hvjdWy"
                  fill="currentColor"
                  focusable="false"
                  height="20"
                  viewBox="0 0 100 100"
                  width="20"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M85.944 20.189H14.056a2.559 2.559 0 00-2.556 2.557v5.144c0 .237.257.509.467.619l37.786 21.583a.63.63 0 00.318.083.634.634 0 00.324-.088L87.039 28.53c.206-.115.752-.419.957-.559.248-.169.504-.322.504-.625v-4.601a2.559 2.559 0 00-2.556-2.556zm2.237 15.457a.644.644 0 00-.645.004L66.799 47.851a.637.637 0 00-.145.985l20.74 22.357a.632.632 0 00.467.204.64.64 0 00.639-.639V36.201a.638.638 0 00-.319-.555zM60.823 51.948a.636.636 0 00-.791-.118l-8.312 4.891a3.243 3.243 0 01-3.208.021l-7.315-4.179a.64.64 0 00-.751.086L12.668 78.415a.64.64 0 00.114 1.02c.432.254.849.375 1.273.375h71.153a.635.635 0 00.585-.385.635.635 0 00-.118-.689L60.823 51.948zm-26.489-2.347a.639.639 0 00-.115-1.023L12.453 36.146a.638.638 0 00-.953.556v32.62a.637.637 0 001.073.468l21.761-20.189z"
                  />
                </svg>
              </a>
            </nav>
          </div>
          <div
            class="col-6 col-sm-4 col-md-2 offset-lg-2"
          >
            <h2
              class="footer__NavHeading-htpw4v-1 hljzCu"
            >
              About
            </h2>
            <nav>
              <a
                class="link__StyledLink-sc-1m1n17q-0 diqghp footer__FooterLink-htpw4v-3 footer__NavLink-htpw4v-4 cILDXv"
                href="/news-and-updates/0x-tracker"
              >
                Blog
              </a>
              <a
                class="link__StyledLink-sc-1m1n17q-0 link___StyledStyledLink-sc-1m1n17q-1 jcGDtB footer__FooterLink-htpw4v-3 footer__NavLink-htpw4v-4 cILDXv"
                href="https://0xtracker.typeform.com/to/HfuuTK"
                rel="noreferrer noopener"
                target="_blank"
              >
                Contact
              </a>
              <a
                class="link__StyledLink-sc-1m1n17q-0 link___StyledStyledLink-sc-1m1n17q-1 jcGDtB footer__FooterLink-htpw4v-3 footer__NavLink-htpw4v-4 cILDXv"
                href="https://docs.0xtracker.com/faqs"
                rel="noreferrer noopener"
                target="_blank"
              >
                FAQs
              </a>
              <a
                class="link__StyledLink-sc-1m1n17q-0 link___StyledStyledLink-sc-1m1n17q-1 jcGDtB footer__FooterLink-htpw4v-3 footer__NavLink-htpw4v-4 cILDXv"
                href="https://docs.0xtracker.com/"
                rel="noreferrer noopener"
                target="_blank"
              >
                Overview
              </a>
            </nav>
          </div>
          <div
            class="col-6 col-sm-4 col-md-2"
          >
            <h2
              class="footer__NavHeading-htpw4v-1 hljzCu"
            >
              Developers
            </h2>
            <nav>
              <a
                class="link__StyledLink-sc-1m1n17q-0 link___StyledStyledLink-sc-1m1n17q-1 jcGDtB footer__FooterLink-htpw4v-3 footer__NavLink-htpw4v-4 cILDXv"
                href="https://docs.0xtracker.com/api-reference/introduction"
                rel="noreferrer noopener"
                target="_blank"
              >
                API Docs
              </a>
              <a
                class="link__StyledLink-sc-1m1n17q-0 link___StyledStyledLink-sc-1m1n17q-1 jcGDtB footer__FooterLink-htpw4v-3 footer__NavLink-htpw4v-4 cILDXv"
                href="https://docs.0xtracker.com/contributing"
                rel="noreferrer noopener"
                target="_blank"
              >
                Contributing
              </a>
              <a
                class="link__StyledLink-sc-1m1n17q-0 link___StyledStyledLink-sc-1m1n17q-1 jcGDtB footer__FooterLink-htpw4v-3 footer__NavLink-htpw4v-4 cILDXv"
                href="https://github.com/0xtracker"
                rel="noreferrer noopener"
                target="_blank"
              >
                GitHub
              </a>
            </nav>
          </div>
          <div
            class="col-6 col-sm-4 col-md-3 col-lg-2"
          >
            <h2
              class="footer__NavHeading-htpw4v-1 hljzCu"
            >
              Resources
            </h2>
            <nav>
              <a
                class="link__StyledLink-sc-1m1n17q-0 link___StyledStyledLink-sc-1m1n17q-1 jcGDtB footer__FooterLink-htpw4v-3 footer__NavLink-htpw4v-4 cILDXv"
                href="https://www.buymeacoffee.com/0xTracker"
                rel="noreferrer noopener"
                target="_blank"
              >
                Donate
              </a>
              <a
                class="link__StyledLink-sc-1m1n17q-0 diqghp footer__FooterLink-htpw4v-3 footer__NavLink-htpw4v-4 cILDXv"
                href="/privacy"
              >
                Privacy Policy
              </a>
              <a
                class="link__StyledLink-sc-1m1n17q-0 link___StyledStyledLink-sc-1m1n17q-1 jcGDtB footer__FooterLink-htpw4v-3 footer__NavLink-htpw4v-4 cILDXv"
                href="https://trello.com/b/4cY2eOsQ/0x-tracker-roadmap"
                rel="noreferrer noopener"
                target="_blank"
              >
                Roadmap
              </a>
              <a
                class="link__StyledLink-sc-1m1n17q-0 diqghp footer__FooterLink-htpw4v-3 footer__NavLink-htpw4v-4 cILDXv"
                href="/terms"
              >
                Terms Of Use
              </a>
            </nav>
          </div>
        </div>
        <div
          class="footer__SecondaryLinks-htpw4v-8 dNNsEr"
        >
          <div
            class="footer__License-htpw4v-7 kYdoUS"
          >
            Licensed under
             
            <a
              class="link__StyledLink-sc-1m1n17q-0 link___StyledStyledLink-sc-1m1n17q-1 jcGDtB footer__FooterLink-htpw4v-3 dTeeTl"
              href="https://github.com/0xTracker/0x-tracker-client/blob/master/LICENSE"
              rel="noreferrer noopener"
              target="_blank"
            >
              Apache-2.0
            </a>
          </div>
          <div
            class="footer__PoweredBy-htpw4v-6 erSKNj"
          >
            Powered by 
            <a
              class="link__StyledLink-sc-1m1n17q-0 link___StyledStyledLink-sc-1m1n17q-1 jcGDtB footer__FooterLink-htpw4v-3 dTeeTl"
              href="https://0x.org"
              rel="noreferrer noopener"
              target="_blank"
            >
              0x
            </a>
             and
             
            <a
              class="link__StyledLink-sc-1m1n17q-0 link___StyledStyledLink-sc-1m1n17q-1 jcGDtB footer__FooterLink-htpw4v-3 dTeeTl"
              href="https://ethereum.org/"
              rel="noreferrer noopener"
              target="_blank"
            >
              Ethereum
            </a>
             with pricing data from
             
            <a
              class="link__StyledLink-sc-1m1n17q-0 link___StyledStyledLink-sc-1m1n17q-1 jcGDtB footer__FooterLink-htpw4v-3 dTeeTl"
              href="https://min-api.cryptocompare.com/"
              rel="noreferrer noopener"
              target="_blank"
            >
              CryptoCompare
            </a>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
